include/exadg/structure/spatial_discretization/operators/nonlinear_operator.cpp

```diff
@@ -326,20 +324,21 @@ NonLinearOperator<dim, Number>::do_cell_integral(IntegratorCell & integrator) co
     // kinematics
     tensor const Grad_delta = integrator.get_gradient(q);
 
-    tensor const F_lin = get_F<dim, Number>(integrator_lin->get_gradient(q));
+    tensor const Grad_displacement = integrator_lin->get_gradient(q);
```

Comment:
minor comment: `Grad_lin` or `Grad_d_lin` would be shorter and in line with the other variable names `F_lin`, `S_lin`.